internal/admission/handler.go

```diff
@@ -4,13 +4,13 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"net/http"
-
 	"github.com/go-logr/logr"
+	"github.com/kong/kubernetes-ingress-controller/v3/internal/metrics"
```

Comment:
Please move this import item to the last section for packages from the repo itself.